docs/topics/migrations.txt

```diff
@@ -622,26 +622,136 @@ to be able to write your own, see the :doc:`migration operations reference
 </ref/migration-operations>` and the "how-to" on :doc:`writing migrations
 </howto/writing-migrations>`.
 
-.. _migration-squashing:
+.. _migration-trimming:
 
-Squashing migrations
-====================
+Trimming migrations
+===================
 
 You are encouraged to make migrations freely and not worry about how many you
 have; the migration code is optimized to deal with hundreds at a time without
 much slowdown. However, eventually you will want to move back from having
-several hundred migrations to just a few, and that's where squashing comes in.
+several hundred migrations to just a few. That's where trimming migrations
+comes in.
 
-Squashing is the act of reducing an existing set of many migrations down to
+Trimming is the act of reducing an existing set of many migrations down to
 one (or sometimes a few) migrations which still represent the same changes.
 
-Django does this by taking all of your existing migrations, extracting their
-``Operation``\s and putting them all in sequence, and then running an optimizer
-over them to try and reduce the length of the list - for example, it knows
-that :class:`~django.db.migrations.operations.CreateModel` and
-:class:`~django.db.migrations.operations.DeleteModel` cancel each other out,
-and it knows that :class:`~django.db.migrations.operations.AddField` can be
-rolled into :class:`~django.db.migrations.operations.CreateModel`.
+There are two common approaches to trimming migrations. The first is to use the
+``squashmigrations`` command to create a merged migration file, however this
+approach can be impractical on complex projects. The second approach is to
+coordinate with your team to ensure that all installations of your app are up
+to date, then to have a scheduled time when migrations are removed and
+recreated from scratch.
```

Comment:
I don't think this is what you want to say. You say there are **two ways** `squashmigrations` and _Nuke 'em™_ but then immediately go on to add a new section _Trimming migrations manually_, which is a **third** 🤔

Reply:
The idea I had here was that "Trimming" refers to the task of somehow removing migrations, and that squashing is one approach to trimming, and deleting is the other.

Reply:
I see, sorry, yes, you're describing the _Nuke 'em_ policy. 

I'd call this something like "Recreating" or "Resetting". 🤔